pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "p5"
+version = "0.8.4"
+description = "p5 is a Python library that provides high level drawing functionality to help you quickly create simulations and interactive art using Python"
+authors = ["Your Name <you@example.com>"]
```

Comment:
You can find the author here - https://github.com/p5py/p5/blob/master/p5/__version__.py#L23